src/client.cpp

```diff
@@ -305,15 +335,27 @@ void client_statistics(int serverNo, Message *pMsgRequest) {
 
         TicksDuration avgRtt = counter ? sumRtt / (int)counter : TicksDuration::TICKS0;
         TicksDuration avgLatency = avgRtt / 2;
-        double usecAvarage =
-            g_pApp->m_const_params.full_rtt ? avgRtt.toDecimalUsec() : avgLatency.toDecimalUsec();
-
         TicksDuration stdDev = TicksDuration::stdDev(pLat, counter);
-        log_msg_file2(f, MAGNETA "====> avg-%s=%.3lf (std-dev=%.3lf)" ENDCOLOR,
-                      round_trip_str[g_pApp->m_const_params.full_rtt], usecAvarage,
-                      stdDev.toDecimalUsec());
+        TicksDuration mad = TicksDuration::mad(pLat, counter);
+        TicksDuration medianad = TicksDuration::medianad(pLat, counter);
+        TicksDuration siqr = TicksDuration::siqr(pLat, counter);
+        TicksDuration *sortedpLat = &pLat[0]; // alias for pLat after being sorted
```

Comment:
POD; Not obvious where latency list is sorted. Move sorting from inside `medianad` function to beginning of `client_statistics`.